src/useQuery.ts

```diff
@@ -163,6 +165,23 @@ export function useQuery<
         };
       }
 
+      // Taken from https://github.com/apollographql/react-apollo/blob/22f8ebf52b26b348d6be905d5b7fbbfea51c1541/src/Query.tsx#L455-L472
+      if (
+        partialRefetch &&
+        Object.keys(result.data).length === 0 &&
+        result.partial &&
+        fetchPolicy !== 'cache-only'
+      ) {
+        const partialRefetchResult = {
+          ...result.data,
```

Comment:
These are empty so maybe it would be better, to keep stale data from previous result using `useRef`.

Reply:
or maybe handle this here in a similar way as error is https://github.com/trojanowski/react-apollo-hooks/pull/159/files#diff-9affe4a75e3f039c2a00e27c8bf815aeR148

Reply:
BTW, sorry to invade this PR without introduction. I didn't catch it in time and coded basically the same thing yesterday evening. So now I'm just sharing my experience. Thanks for the PR!

Reply:
Thanks for the comments! I'll update the PR with this! is much better.

Reply:
FWIW Also this one is related to the need of showing stale data:
https://github.com/trojanowski/react-apollo-hooks/pull/134